### ISSUE TITLE
Add block id asciidoc reference for code example in alternative_report.json

### ISSUE THREAD

**Issue by ezimuel** [elastic/docs#1721]
I'm building a report to track the code examples for each client repositories (see https://github.com/elastic/clients-team/pull/114). I would like to have the link to the Elastic documentation where the code example is available. Now, we have the anchor link with the digest (thanks to https://github.com/elastic/docs/pull/1717) but I don't have the HTML page name.
I'm using the [alternatives_report.json](https://raw.githubusercontent.com/elastic/built-docs/master/raw/en/elasticsearch/reference/master/alternatives_report.json) as source and we have only the asciidoc file for each code example. We need also the asciidoc **block id** to find out the HTML page name.

For instance, this code example [getting-started-index.html#3f3b3e207f79303ce6f86e03e928e062](https://www.elastic.co/guide/en/elasticsearch/reference/master/getting-started-index.html#3f3b3e207f79303ce6f86e03e928e062)
is located in `getting-started.asciidoc` but the URL of the HTML page is `getting-started-index.html`. In order to get this URL I need the **`[[getting-started-index]]`** block id in the `alternative_report.json` for the code example. Can we add this block name in the `source_location`? 

For instance, something like this:
```json
{
    "source_location": {
        "file": "getting-started.asciidoc",
        "block": "getting-started-index",
        "line": 253,
    },
    "digest":"3f3b3e207f79303ce6f86e03e928e062",
}
```

Thoughts?

CC @elastic/es-clients-core

**Comment by nik9000** [comment]
This *should* be possible, with our migration away from docbook. But it is a bit more difficult than you'd expect.

**Comment by gtback** [comment]
Yeah, we don't enforce a correspondence between the input asciidoc file and the output HTML chunk. I'll have to look into how easy it would be for the chunker to add this information to the alternatives report.
